shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -127,19 +127,29 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       value = pair[1];
 
       if (tableName != null) {
-        if (!Property.isValidTablePropertyKey(property)) {
-          throw new BadArgumentException("Invalid per-table property.", fullCommand,
-              fullCommand.indexOf(property));
+        if (!Property.isTablePropertyValid(property, value)) {
+          if (!Property.isValidTablePropertyKey(property)) {
+            throw new BadArgumentException("Invalid per-table property.", fullCommand,
```

Comment:
This exception message could mention that the prop key specifically is invalid.

Reply:
Added property value to the exception message
https://github.com/apache/accumulo/pull/3146/files#diff-dedfd7e340f58e41d93da9c80014542cb08d27c4448d77cfcf69ff85457a0f0cR351